src/components/QRCode/QRCodeWithTheme.js

```diff
@@ -39,14 +38,13 @@ const QRCodeWithTheme = (props: Props) => {
     getRef,
   } = props;
   const colors = getThemeColors(theme);
-  const themeType = getThemeType(theme);
   return (
     <QRCode
       getRef={getRef}
       value={value}
       size={size}
-      color={themeType === DARK_THEME ? '#FFFFFF' : colors.text}
-      backgroundColor={colors.card}
+      color={getColorByThemeOutsideStyled(theme.current, { lightKey: 'basic010', darkCustom: '#FFFFFF' })}
```

Comment:
The qr code in dark mode has reversed black and white colors? Can't the dark mode be just normal?

![Screenshot 2020-10-22 at 00 47 02](https://user-images.githubusercontent.com/6280457/96797059-1b916100-1400-11eb-9ec4-581e70a234ae.png)

![Screenshot 2020-10-22 at 00 46 11](https://user-images.githubusercontent.com/6280457/96796545-fdc3fc00-13ff-11eb-879f-29a122e09326.png)

Reply:
Yeah, it can :)
I'll update it